app/models/patient.rb

```diff
@@ -105,23 +104,6 @@ class Patient < ApplicationRecord
   validates_associated :address, if: :address
   validates_associated :phone_numbers, if: :phone_numbers
 
-  def self.follow_ups_with(model_name, period, time_column: "recorded_at", at_region: nil, current: true, last: nil)
```

Comment:
I was struggling with figuring out how to handle follow ups, and I realized extracting to its own object would be the best way. Ideally the api for these sorts of queries will be much more constrained in the future, but right now too many code paths access the scopes above to remove them.